src/main/java/org/jabref/logic/importer/fileformat/RisImporter.java

```diff
@@ -27,6 +27,8 @@
 
     private static final Pattern RECOGNIZED_FORMAT_PATTERN = Pattern.compile("TY  - .*");
     private static DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy");
+    //stores all the date tags from highest to lowest priority
+    private final String[] dateTags = {"Y1", "PY", "DA", "Y2"};
```

Comment:
I would use a List here  instead of an array. e.g. List<String> datTages = Arrays-asList()

Reply:
Yes, that's way better, changed to List<String>.

Reply:
Yes, that's way better, changed to List<String>